src/sentry/snuba/metrics_layer/query.py

```diff
@@ -343,10 +343,6 @@ def _resolve_timeseries_metadata(
     else:
         mappings[metric.mri] = metric.id
 
-    if not metric.entity:
-        entity = _resolve_metrics_entity(metric.mri)  # This should eventually be done in Snuba
-        metric = metric.set_entity(entity.value)
```

Comment:
@iambriccardo are the code removals in here intentional?

I see that the code was added by someone else.

Reply:
Yes this is intended. The new version of the SDK doesn't use the `entity`.